contracts/staking/contracts/src/staking_pools/MixinCumulativeRewards.sol

```diff
@@ -264,59 +260,108 @@ contract MixinCumulativeRewards is
     )
         internal
         view
-        returns (uint256)
+        returns (uint256 reward)
     {
-        // sanity check inputs
         if (memberStakeOverInterval == 0) {
             return 0;
         }
 
-        // sanity check interval
-        if (beginEpoch >= endEpoch) {
+        // Sanity check interval
+        if (beginEpoch > endEpoch) {
             LibRichErrors.rrevert(
                 LibStakingRichErrors.CumulativeRewardIntervalError(
-                    LibStakingRichErrors.CumulativeRewardIntervalErrorCode.BeginEpochMustBeLessThanEndEpoch,
+                    LibStakingRichErrors
+                        .CumulativeRewardIntervalErrorCode
+                        .BeginEpochMustBeLessThanEndEpoch,
                     poolId,
                     beginEpoch,
                     endEpoch
                 )
             );
         }
 
-        // sanity check begin reward
-        IStructs.Fraction memory beginReward = _cumulativeRewardsByPool[poolId][beginEpoch];
+        // Sanity check begin reward
+        IStructs.Fraction memory beginReward =
+            _cumulativeRewardsByPool[poolId][beginEpoch];
```

Comment:
I gotta say, I think whatever formatter you're using is a bit aggressive on the line lengths. Might be hard to undo this though 🤷‍♂